src/pages/public_registration/action_accordions/DoiRequestAccordion.tsx

```diff
@@ -252,6 +260,25 @@ export const DoiRequestAccordion = ({
               disabled={isLoadingData || isLoading !== LoadingState.None}>
               {t('common.reject_doi')}
             </LoadingButton>
+            <ConfirmDialog
+              open={showConfirmDialogAssignDoi}
+              title={t('registration.public_page.tasks_panel.no_published_files_on_registration')}
+              onAccept={async () => {
+                await ticketMutation.mutateAsync({ status: 'Completed' });
+                toggleConfirmDialogAssignDoi();
+              }}
+              isLoading={isLoadingData || ticketMutation.isPending}
+              onCancel={() => toggleConfirmDialogAssignDoi()}>
+              <Trans
+                t={t}
+                i18nKey="registration.public_page.tasks_panel.no_published_files_on_registration_description"
+                components={[
+                  <Typography paragraph key="1" />,
+                  <Typography paragraph key="2" />,
+                  <Typography paragraph key="3" />,
+                ]}
+              />
```

Comment:
Tror ikke du trenger en separat `<Typography />` for hvert avsnitt🤔 Virker som det er greit at alle har `key="1"` 

```suggestion
                components={[<Typography paragraph key="1" />]}
              />
```